### PR TITLE
ECO-3470-Implemented-Asset-Sidebar-Screen

### PR DESCRIPTION
**Pull request by Contentstack-AnkitaD**
- Implemented the UI for Asset Sidebar Location:
image

### CODE REVIEW

**Review by MaitriGurey1 — APPROVED** [review]

File: src/containers/AssetSidebarWidget/AssetSidebar.tsx [hunk 1]
```diff
@@ -1,23 +1,64 @@
...
 import localeTexts from "../../common/locales/en-us/index";
 import parse from "html-react-parser";
+import { useAppConfig } from "../../common/hooks/useAppConfig";
+import "../index.css";
+import "./AssetSidebar.css";
+import Icon from "../../assets/Asset-Sidebar-Logo.svg";
+import ReadOnly from "../../assets/lock.svg";
+//TODO:  need to update the CTA button
+import Copy from "../../assets/Copy.svg";
+//TODO:  need to update the RC modal
```
Comment (abhishek305) [inline src/containers/AssetSidebarWidget/AssetSidebar.tsx:11]: do we need these comments? If not remove this

File: src/containers/AssetSidebarWidget/AssetSidebar.tsx [hunk 2]
```diff
@@ -1,23 +1,64 @@
...
+import RawConfigModal from "../../components/ViewRawConfig/RawConfigModal";
 
 const AssetSidebarExtension = () => {
+  const appConfig = useAppConfig();
+
+  const [isRawConfigModalOpen, setRawConfigModalOpen] = useState(false);
```
Comment (abhishek305) [inline src/containers/AssetSidebarWidget/AssetSidebar.tsx:17]: add proper type for this state

File: src/containers/AssetSidebarWidget/AssetSidebar.tsx [hunk 3]
```diff
@@ -1,23 +1,64 @@
...
+
+  const handleViewRawConfig = () => {
+    setRawConfigModalOpen(true);
+  };
+
+  const handleCloseModal = () => {
+    setRawConfigModalOpen(false);
+  };
```
Comment (abhishek305) [inline src/containers/AssetSidebarWidget/AssetSidebar.tsx:25]: wrap this in useCallback

File: src/containers/index.css [hunk 4]
```diff
@@ -78,7 +110,7 @@
 .info {
   color: var(--text-secondary-text-purple-gray, #6e6b86);
   font-family: Inter;
-  font-size: 0.875rem;
+  /* font-size: 0.875rem; */
```
Comment (abhishek305) [inline src/containers/index.css:113]: remove comment

**Review by abhishek305 — CHANGES_REQUESTED** [review]

**Review by Amitkanswal — APPROVED** [review]

**Review by abhishek305 — APPROVED** [review]

**Pull request merged**